kie-server-parent/kie-server-controller/kie-server-controller-websocket/src/main/java/org/kie/server/controller/websocket/notification/WebSocketKieServerControllerNotification.java

```diff
@@ -41,9 +41,7 @@ public void configure() {
     public void onNotificationClientConnect(final Session session) {
         LOGGER.debug("New Web Socket Notification Client session: {}",
                      session.getId());
-        synchronized (manager) {
```

Comment:
If I am not mistaken we added it here due to some race conditions discovered via cloud testing, @sutaakar am I anywhere close with this...?

Reply:
It was fixing JBPM-6581, the case when several Kie servers connect/disconnect to the controller at once.

Reply:
@mswiderski I found that it wasn't needed to sync in here as both add and remove session methods are backed by a concurrent hash map.